src/js/media-container.ts

```diff
@@ -169,7 +169,7 @@ template.innerHTML = /*html*/ `
       Attributes.NO_AUTOHIDE
     }]):not([role=dialog])) {
       opacity: 0;
-      transition: opacity 1s;
+      transition: var(--media-hide-transition, opacity 1s);
```

Comment:
this is good but I would change the naming to include control. we've also used `in` and `out` naming for transitions to make that distinction. @heff @cjpillsbury thoughts?

I propose making this CSS var `--media-control-transition-out` and then the transition above we can make `--media-control-transition-in`

Reply:
Great! I renamed the variable and created the variable for the show transition